### ISSUE TITLE
The recognition result becomes very bad after fine tune

### ISSUE THREAD

**Issue by Gnakkk** [tesseract-ocr/tesseract#3850]
### Environment
Tesseract Version: v5.1.0.20220510

### Current Behavior:
Iteration 1199: GROUND  TRUTH : 在谁都没有注意的时候,
File D:\tr\chi.chi.exp0.lstmf line 0 (Perfect):
Mean rms=0.192%, delta=0.009%, train=0%(0%), skip ratio=0%
2 Percent improvement time=0, best error was 4.66 @ 194
At iteration 194/1200/1200, Mean rms=0.192000%, delta=0.009000%, BCER train=0.000000%, BWER train=0.000000%, skip ratio=0.000000%,  New best BCER = 0.000000 wrote best model:D:\tr\trainingoutput/_0.000000_194_1200.checkpoint wrote checkpoint.

Finished! Selected model with minimal training error rate (BCER) = 0
Loaded file D:\tr\trainingoutput/_checkpoint, unpacking...

original text： 在谁都没有注意的时候，

tesseract chi.chi.exp0.png stdout -l chi_sim --psm 6:
official chi_sim.traineddata results: 
在谁都没有注意的时候， 和SS

tesseract chi.chi.exp0.png stdout -l chi_sim_ft --psm 6:
fine tuned chi_sim.traineddata results: 
看 入 注 昌 候 SN
的 SS

The recognition result becomes very bad after fine tune. why ?

### Expected Behavior:
At least the original training image can be recognized correctly.

**Comment by stweil** [comment]
The finetuned model only includes the neural network part. Maybe adding dictionary, configuration etc. from the original model fixes the issue.

**Comment by amitdo** [comment]
So you trained just one line?

This is not the way you're supposed to fine tune a model.

**Comment by Gnakkk** [comment]
So what should I do?

**Comment by amitdo** [comment]
Train with all the characters/glyphs that were used in the original traineddata, +/- a few more/less. For chi_sim, you'll need thousands of lines for a single font fine tuning. Lines should contain in average 50-60 glyphs.

The accuracy might still be worst than the generic model.

For questions, please use our forum.
